go-controller/pkg/ovn/ovn.go

```diff
@@ -769,6 +795,11 @@ func (oc *Controller) WatchNamespaces() {
 		},
 	}, oc.syncNamespaces)
 	klog.Infof("Bootstrapping existing namespaces and cleaning stale namespaces took %v", time.Since(start))
+	// TODO(FF): make this function return failures
```

Comment:
why cant this return an error?

Reply:
It is, but I did on on a separate commit. Will squash it to make it less confusing.